RELEASE_NOTES.rst

```diff
@@ -1,8 +1,8 @@
 coala-bears 0.2.0
 =================
 
-In this release, coala-bears has been revamped with new bears, more configs for
-existing bears.
+In this release, coala-bears has been revamped with new bears and more
+configs for existing bears.
```

Comment:
this doesn't belong into this commit (https://github.com/coala-analyzer/coala-bears/pull/385/commits/6df585fc426b981396dc3b1a9bf5fb197a694a08)